src/vue/common/OperationRecord.vue

```diff
@@ -0,0 +1,215 @@
+<template>
+  <div class="app__card operation-record">
+    <table class="operation-record__table">
+      <tr class="operation-record__tr">
+        <td>
+          <span class="operation-record__date">
+            {{ operation.date | formatCalendar }}
+          </span>
+        </td>
+        <td>
+          {{ `operation-names.${operation.typeI}` | globalize }}
+        </td>
+        <td>
+          <span
+            :class="{'operation-record__negative-amount'
+              : accountId === operation.sender}">
+            {{
+              { value: operation.amount, currency: operation.asset } |
+                formatMoney
+            }}
+          </span>
+        </td>
+        <td class="operation-record__counterparty">
+          {{ operation.counterparty }}
+        </td>
+        <td class="operation-record__status">
+          <span :class="`operation-record__status--${operation.state}`">
+            {{ operation.state }}
+          </span>
+        </td>
+        <td class="operation-record__td-btn">
+          <button
+            class="operation-record__btn"
+            @click="isOpenDetails = !isOpenDetails">
+            <i
+              v-if="isOpenDetails"
+              class="mdi mdi-chevron-up" />
+            <i
+              v-else
+              class="mdi mdi-chevron-down" />
+          </button>
+        </td>
+      </tr>
+    </table>
+    <template v-if="isOpenDetails">
+      <details-issuance
+        v-if="OP_TYPES.createIssuanceRequest === operation.typeI"
+        :operation="operation" />
+      <payment-details
+        v-if="OP_TYPES.paymentV2 === operation.typeI ||
+          OP_TYPES.payment === operation.typeI"
+        :operation="operation" />
+      <match-details
+        v-if="OP_TYPES.manageOffer === operation.typeI"
+        :operation="operation" />
+      <match-details
+        v-if="OP_TYPES.checkSaleState === operation.typeI"
+        :operation="operation" />
+      <withdrawal-details
+        v-if="OP_TYPES.createWithdrawalRequest === operation.typeI"
```

Comment:
do not compare like CONST === VAR, compare VAR === CONST instead, the current way looks like you’re master Yoda